Lerna/pub beta

zhenlan-zhao opened the pull request:
Merge lerna/pub-beta into lerna/pub

zhenlan-zhao committed:
feat: init @alicloud/console-fetcher-interceptor-res-risk-new & @alicloud/u2f-api

zhenlan-zhao committed:
feat: 修改 @alicloud/console-fetcher-interceptor-res-risk，加入新版主子账号风控

zhenlan-zhao committed:
feat: 删除 alicloud/console-fetcher-interceptor-res-risk-new & 修改 fetcher-demo-helper

zhenlan-zhao committed:
fix: 修复 Code Review 错误

zhenlan-zhao committed:
fix: 修复 Code Review 的问题

zhenlan-zhao committed:
fix: 去除不必要的字体设置

zhenlan-zhao committed:
feat: @alicloud/console-fetcher-interceptor-res-risk 加入 @alicloud/u2f-api 作为依赖

zhenlan-zhao committed:
feat: 解决 package.json 的冲突

zhenlan-zhao committed:
Merge branch 'lerna/pub-prerelease' into feat/1.1.0

justnewbee committed:
Merge pull request #8 from zhenlan-zhao/feat/1.1.0

FEAT new risk for fetcher from zzl

Commit:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.5-alpha.0
 - @alicloud/console-base-global@1.1.66-alpha.0
 - @alicloud/console-base-log-sls@1.2.1-alpha.0
 - @alicloud/console-base-messenger@1.3.57-alpha.0
 - @alicloud/console-fetcher-basic@1.5.31-alpha.0
 - @alicloud/console-fetcher-interceptor-arms@1.1.53-alpha.0
 - @alicloud/console-fetcher-interceptor-fecs@1.1.63-alpha.0
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.59-alpha.0
 - @alicloud/console-fetcher-interceptor-req-security@1.1.53-alpha.0
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.53-alpha.0
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.55-alpha.0
 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.0
 - @alicloud/console-fetcher-interceptor-sls@1.1.62-alpha.0
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.0
 - @alicloud/console-fetcher@1.5.42-alpha.0
 - @alicloud/console-logger-sls@1.2.1-alpha.0
 - @alicloud/fetcher-demo-helpers@1.1.50-alpha.0
 - @alicloud/fetcher-fetch@1.2.50-alpha.0
 - @alicloud/fetcher-interceptor-cache-local@1.1.27-alpha.0
 - @alicloud/fetcher-interceptor-merger@1.0.43-alpha.0
 - @alicloud/fetcher-jsonp@1.1.51-alpha.0
 - @alicloud/fetcher@1.1.53-alpha.0
 - @alicloud/logger-sls@1.5.1-alpha.0
 - @alicloud/u2f-api@0.0.1-alpha.0

Commit:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.5-alpha.1
 - @alicloud/console-base-global@1.1.66-alpha.1
 - @alicloud/console-base-log-sls@1.2.1-alpha.1
 - @alicloud/console-base-messenger@1.3.57-alpha.1
 - @alicloud/console-fetcher-basic@1.5.31-alpha.1
 - @alicloud/console-fetcher-interceptor-arms@1.1.53-alpha.1
 - @alicloud/console-fetcher-interceptor-fecs@1.1.63-alpha.1
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.59-alpha.1
 - @alicloud/console-fetcher-interceptor-req-security@1.1.53-alpha.1
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.53-alpha.1
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.55-alpha.1
 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.1
 - @alicloud/console-fetcher-interceptor-sls@1.1.62-alpha.1
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.1
 - @alicloud/console-fetcher@1.5.42-alpha.1
 - @alicloud/console-logger-sls@1.2.1-alpha.1
 - @alicloud/fetcher-demo-helpers@1.1.50-alpha.1
 - @alicloud/fetcher-fetch@1.2.50-alpha.1
 - @alicloud/fetcher-interceptor-cache-local@1.1.27-alpha.1
 - @alicloud/fetcher-interceptor-merger@1.0.43-alpha.1
 - @alicloud/fetcher-jsonp@1.1.51-alpha.1
 - @alicloud/fetcher@1.1.53-alpha.1
 - @alicloud/logger-sls@1.5.1-alpha.1
 - @alicloud/u2f-api@0.0.1-alpha.1

Commit:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.5-alpha.2
 - @alicloud/console-base-global@1.1.66-alpha.2
 - @alicloud/console-base-log-sls@1.2.1-alpha.2
 - @alicloud/console-base-messenger@1.3.57-alpha.2
 - @alicloud/console-fetcher-basic@1.5.31-alpha.2
 - @alicloud/console-fetcher-interceptor-arms@1.1.53-alpha.2
 - @alicloud/console-fetcher-interceptor-fecs@1.1.63-alpha.2
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.59-alpha.2
 - @alicloud/console-fetcher-interceptor-req-security@1.1.53-alpha.2
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.53-alpha.2
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.55-alpha.2
 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.2
 - @alicloud/console-fetcher-interceptor-sls@1.1.62-alpha.2
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.2
 - @alicloud/console-fetcher@1.5.42-alpha.2
 - @alicloud/console-logger-sls@1.2.1-alpha.2
 - @alicloud/fetcher-demo-helpers@1.1.50-alpha.2
 - @alicloud/fetcher-fetch@1.2.50-alpha.2
 - @alicloud/fetcher-interceptor-cache-local@1.1.27-alpha.2
 - @alicloud/fetcher-interceptor-merger@1.0.43-alpha.2
 - @alicloud/fetcher-jsonp@1.1.51-alpha.2
 - @alicloud/fetcher@1.1.53-alpha.2
 - @alicloud/logger-sls@1.5.1-alpha.2
 - @alicloud/u2f-api@0.0.1-alpha.2

zhenlan-zhao committed:
feat: res-risk 加入@types/qrcode.react

zhenlan-zhao committed:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.5-alpha.3
 - @alicloud/console-base-global@1.1.66-alpha.3
 - @alicloud/console-base-log-sls@1.2.1-alpha.3
 - @alicloud/console-base-messenger@1.3.57-alpha.3
 - @alicloud/console-fetcher-basic@1.5.31-alpha.3
 - @alicloud/console-fetcher-interceptor-arms@1.1.53-alpha.3
 - @alicloud/console-fetcher-interceptor-fecs@1.1.63-alpha.3
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.59-alpha.3
 - @alicloud/console-fetcher-interceptor-req-security@1.1.53-alpha.3
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.53-alpha.3
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.55-alpha.3
 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.3
 - @alicloud/console-fetcher-interceptor-sls@1.1.62-alpha.3
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.3
 - @alicloud/console-fetcher@1.5.42-alpha.3
 - @alicloud/console-logger-sls@1.2.1-alpha.3
 - @alicloud/fetcher-demo-helpers@1.1.50-alpha.3
 - @alicloud/fetcher-fetch@1.2.50-alpha.3
 - @alicloud/fetcher-interceptor-cache-local@1.1.27-alpha.3
 - @alicloud/fetcher-interceptor-merger@1.0.43-alpha.3
 - @alicloud/fetcher-jsonp@1.1.51-alpha.3
 - @alicloud/fetcher@1.1.53-alpha.3
 - @alicloud/logger-sls@1.5.1-alpha.3
 - @alicloud/u2f-api@0.0.1-alpha.3

zhenlan-zhao committed:
feat: res-risk 中不应该指定 @alicloud/u2f-api 的版本为0.0.1

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.4
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.4
 - @alicloud/console-fetcher@1.5.42-alpha.4

zhenlan-zhao committed:
feat: res-risk @alicloud/u2f-api 的版本是 0.0.1-alpha.3

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.5
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.5
 - @alicloud/console-fetcher@1.5.42-alpha.5

zhenlan-zhao committed:
feat: 加上调试信息

zhenlan-zhao committed:
feat: 加上调试信息

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.6
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.6
 - @alicloud/console-fetcher@1.5.42-alpha.6

zhenlan-zhao committed:
feat: new mfa auth & new mfa choose 问题修复

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.7
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.7
 - @alicloud/console-fetcher@1.5.42-alpha.7

zhenlan-zhao committed:
feat: 修改 readme

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.8
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.8
 - @alicloud/console-fetcher@1.5.42-alpha.8

zhenlan-zhao committed:
feat: 身份核验 Content-Type 修改为 application/json & mfa choose 页面调整

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.9
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.9
 - @alicloud/console-fetcher@1.5.42-alpha.9

zhenlan-zhao committed:
feat: getMfaInfoToBind 接口 response 处理逻辑修改 & 优化 u2f 提示

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.10
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.10
 - @alicloud/console-fetcher@1.5.42-alpha.10

zhenlan-zhao committed:
CHORE: 子账号核身接口请求参数修改 & 弹框样式及行为调整

zhenlan-zhao committed:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.5-alpha.4
 - @alicloud/console-base-global@1.1.66-alpha.4
 - @alicloud/console-base-log-sls@1.2.1-alpha.4
 - @alicloud/console-base-messenger@1.3.57-alpha.4
 - @alicloud/console-fetcher-basic@1.5.31-alpha.4
 - @alicloud/console-fetcher-interceptor-arms@1.1.53-alpha.4
 - @alicloud/console-fetcher-interceptor-fecs@1.1.63-alpha.4
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.59-alpha.4
 - @alicloud/console-fetcher-interceptor-req-security@1.1.53-alpha.4
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.53-alpha.4
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.55-alpha.4
 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.11
 - @alicloud/console-fetcher-interceptor-sls@1.1.62-alpha.4
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.11
 - @alicloud/console-fetcher@1.5.42-alpha.11
 - @alicloud/console-logger-sls@1.2.1-alpha.4
 - @alicloud/fetcher-demo-helpers@1.1.50-alpha.4
 - @alicloud/fetcher-fetch@1.2.50-alpha.4
 - @alicloud/fetcher-interceptor-cache-local@1.1.27-alpha.4
 - @alicloud/fetcher-interceptor-merger@1.0.43-alpha.4
 - @alicloud/fetcher-jsonp@1.1.51-alpha.4
 - @alicloud/fetcher@1.1.53-alpha.4
 - @alicloud/logger-sls@1.5.1-alpha.4

zhenlan-zhao committed:
CHORE: verify 接口修复

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.12
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.12
 - @alicloud/console-fetcher@1.5.42-alpha.12

zhenlan-zhao committed:
CHORE：新版主账号风控调整

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.13
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.13
 - @alicloud/console-fetcher@1.5.42-alpha.13

zhenlan-zhao committed:
FIX: 样式调整 & 新版风控标识由 newRisk:true 改为 consoleVersion:2.0

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.14
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.14
 - @alicloud/console-fetcher@1.5.42-alpha.14

zhenlan-zhao committed:
FIX: 样式调整

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.15
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.15
 - @alicloud/console-fetcher@1.5.42-alpha.15

zhenlan-zhao committed:
FIX: 样式调整 & 标题修改 & 逻辑修复

zhenlan-zhao committed:
FIX: 选择绑定的 MFA 设备交互调整

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.16
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.16
 - @alicloud/console-fetcher@1.5.42-alpha.16

zhenlan-zhao committed:
CHORE: 修改子账号风控对 forbidden, unknown 逻辑的处理 & 新增新版子账号风控对 verifyUrl 为空时的提示

zhenlan-zhao committed:
CHORE: 修改统一的提示 Message 的 Icon 样式

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.17
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.17
 - @alicloud/console-fetcher@1.5.42-alpha.17

zhenlan-zhao committed:
CHORE: 修复国际化文案 eslint 错误

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.18
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.18
 - @alicloud/console-fetcher@1.5.42-alpha.18

zhenlan-zhao committed:
FEAT: 调整被风控时对 openAPI 返回数据的处理 & 调整与核身服务交互的逻辑

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.19
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.19
 - @alicloud/console-fetcher@1.5.42-alpha.19

zhenlan-zhao committed:
FEAT: 调整被风控时对 openAPI 返回数据的处理 & 调整与核身服务交互的逻辑

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.20
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.20
 - @alicloud/console-fetcher@1.5.42-alpha.20

zhenlan-zhao committed:
FEAT: lerna 发布失败，增加差异性内容

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.81-alpha.21
 - @alicloud/console-fetcher-proxy@1.4.45-alpha.21
 - @alicloud/console-fetcher@1.5.42-alpha.21

zhenlan-zhao committed:
FEAT: merge lerna/pub

justnewbee committed:
CHORE up

justnewbee committed:
CHORE ncu

justnewbee committed:
CHORE up

justnewbee committed:
CHORE allow publish on lerna/pub-*

justnewbee committed:
Merge branch 'lerna/pub-prerelease' into lerna/pub-beta

justnewbee committed:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.41-alpha.0
 - @alicloud/console-base-global@1.1.97-alpha.0
 - @alicloud/console-base-log-sls@1.2.30-alpha.0
 - @alicloud/console-base-messenger@1.3.88-alpha.0
 - @alicloud/console-fetcher-basic@1.5.61-alpha.0
 - @alicloud/console-fetcher-interceptor-arms@1.1.82-alpha.0
 - @alicloud/console-fetcher-interceptor-fecs@1.1.92-alpha.0
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.88-alpha.0
 - @alicloud/console-fetcher-interceptor-req-security@1.1.82-alpha.0
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.82-alpha.0
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.85-alpha.0
 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.0
 - @alicloud/console-fetcher-interceptor-sls@1.1.91-alpha.0
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.0
 - @alicloud/console-fetcher@1.5.74-alpha.0
 - @alicloud/console-logger-sls@1.2.30-alpha.0
 - @alicloud/fetcher-demo-helpers@1.1.79-alpha.0
 - @alicloud/fetcher-fetch@1.2.79-alpha.0
 - @alicloud/fetcher-interceptor-cache-local@1.1.56-alpha.0
 - @alicloud/fetcher-interceptor-merger@1.0.72-alpha.0
 - @alicloud/fetcher-jsonp@1.1.80-alpha.0
 - @alicloud/fetcher@1.1.82-alpha.0
 - @alicloud/logger-sls@1.5.30-alpha.0
 - @alicloud/u2f-api@0.0.1-alpha.4

justnewbee committed:
CHORE last build faild

zhenlan-zhao committed:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.41-alpha.1
 - @alicloud/console-base-global@1.1.97-alpha.1
 - @alicloud/console-base-log-sls@1.2.30-alpha.1
 - @alicloud/console-base-messenger@1.3.88-alpha.1
 - @alicloud/console-fetcher-basic@1.5.61-alpha.1
 - @alicloud/console-fetcher-interceptor-arms@1.1.82-alpha.1
 - @alicloud/console-fetcher-interceptor-fecs@1.1.92-alpha.1
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.88-alpha.1
 - @alicloud/console-fetcher-interceptor-req-security@1.1.82-alpha.1
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.82-alpha.1
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.85-alpha.1
 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.1
 - @alicloud/console-fetcher-interceptor-sls@1.1.91-alpha.1
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.1
 - @alicloud/console-fetcher@1.5.74-alpha.1
 - @alicloud/console-logger-sls@1.2.30-alpha.1
 - @alicloud/fetcher-demo-helpers@1.1.79-alpha.1
 - @alicloud/fetcher-fetch@1.2.79-alpha.1
 - @alicloud/fetcher-interceptor-cache-local@1.1.56-alpha.1
 - @alicloud/fetcher-interceptor-merger@1.0.72-alpha.1
 - @alicloud/fetcher-jsonp@1.1.80-alpha.1
 - @alicloud/fetcher@1.1.82-alpha.1
 - @alicloud/logger-sls@1.5.30-alpha.1
 - @alicloud/u2f-api@0.0.1-alpha.5

zhenlan-zhao committed:
FEAT: 重新生成 Token 发布

zhenlan-zhao committed:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.41-alpha.2
 - @alicloud/console-base-global@1.1.97-alpha.2
 - @alicloud/console-base-log-sls@1.2.30-alpha.2
 - @alicloud/console-base-messenger@1.3.88-alpha.2
 - @alicloud/console-fetcher-basic@1.5.61-alpha.2
 - @alicloud/console-fetcher-interceptor-arms@1.1.82-alpha.2
 - @alicloud/console-fetcher-interceptor-fecs@1.1.92-alpha.2
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.88-alpha.2
 - @alicloud/console-fetcher-interceptor-req-security@1.1.82-alpha.2
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.82-alpha.2
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.85-alpha.2
 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.2
 - @alicloud/console-fetcher-interceptor-sls@1.1.91-alpha.2
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.2
 - @alicloud/console-fetcher@1.5.74-alpha.2
 - @alicloud/console-logger-sls@1.2.30-alpha.2
 - @alicloud/fetcher-demo-helpers@1.1.79-alpha.2
 - @alicloud/fetcher-fetch@1.2.79-alpha.2
 - @alicloud/fetcher-interceptor-cache-local@1.1.56-alpha.2
 - @alicloud/fetcher-interceptor-merger@1.0.72-alpha.2
 - @alicloud/fetcher-jsonp@1.1.80-alpha.2
 - @alicloud/fetcher@1.1.82-alpha.2
 - @alicloud/logger-sls@1.5.30-alpha.2
 - @alicloud/u2f-api@0.0.1-alpha.6

zhenlan-zhao committed:
FEAT: 子账号风控加入 CodeType 参数 & 增加 U2F 绑定/验证失败后的提示以及重试的能力

zhenlan-zhao committed:
FEAT: 修改 lerna:publish:prerelease 脚本

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.3
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.3
 - @alicloud/console-fetcher@1.5.74-alpha.3

zhenlan-zhao committed:
FEAT: 补充 zh-tw.ts 缺失的文案，修复 ts 错误

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.4
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.4
 - @alicloud/console-fetcher@1.5.74-alpha.4

zhenlan-zhao committed:
获取 u2f 安全密钥报错信息的国际化 & 加入 u2f 安全密钥获取重试机制

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.5
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.5
 - @alicloud/console-fetcher@1.5.74-alpha.5
 - @alicloud/u2f-api@0.0.1-alpha.7

zhenlan-zhao committed:
FIX: U2F 重试机制 UI 调整

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.6
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.6
 - @alicloud/console-fetcher@1.5.74-alpha.6

zhenlan-zhao committed:
FIX: 绑定 U2F，重新调用被风控接口失败时，不允许重试获取 U2F 安全密钥

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.7
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.7
 - @alicloud/console-fetcher@1.5.74-alpha.7

zhenlan-zhao committed:
FIX: 修改 U2F 错误的提示 & 修改获取用户绑定 U2F 设备信息出错时的 UI

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.8
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.8
 - @alicloud/console-fetcher@1.5.74-alpha.8

zhenlan-zhao committed:
FIX: 当绑定 U2F【成功】，但重新请求被风控的接口报错时，重试需要让用户走 U2F 验证

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.9
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.9
 - @alicloud/console-fetcher@1.5.74-alpha.9

zhenlan-zhao committed:
FIX: 当绑定 U2F【成功】，但重新请求被风控的接口报错时，走 U2F 验证逻辑还需要获取 U2F 验证信息

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@1.1.113-alpha.10
 - @alicloud/console-fetcher-proxy@1.4.79-alpha.10
 - @alicloud/console-fetcher@1.5.74-alpha.10

zhenlan-zhao committed:
FEAT: 为了避免 alpha 版本被 latest 版本覆盖，alpha 版本需要有更高的版本号

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.0
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.0
 - @alicloud/console-fetcher@2.0.0-alpha.0

zhenlan-zhao committed:
FEAT: riskInvalid 场景，或者获取用户 MFA 设备类型失败直接调用 open 方法展示弹窗时，点击取消按钮调用 reject 方法使得后续处理走 catch 逻辑

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.1
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.1
 - @alicloud/console-fetcher@2.0.0-alpha.1

zhenlan-zhao committed:
FEAT: 处理新版主账号风控，填入验证码重新请求被风控接口报错时的 UI 展示

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.2
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.2
 - @alicloud/console-fetcher@2.0.0-alpha.2

zhenlan-zhao committed:
FIX: 接口被风控时返回的风控类型有误时，由于采用的是在 await 之后 throw 错误的方式，因此不应该 useUndefinedAsReject

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.3
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.3
 - @alicloud/console-fetcher@2.0.0-alpha.3

zhenlan-zhao committed:
FIX: Code Review 修改

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.4
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.4
 - @alicloud/console-fetcher@2.0.0-alpha.4

zhenlan-zhao committed:
FEAT: code review 修改 & 虚拟 MFA 绑定时密钥展示需要通过点击按钮触发

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.5
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.5
 - @alicloud/console-fetcher@2.0.0-alpha.5

zhenlan-zhao committed:
FIX：绑定 U2F 设备用户返回上一步再禁止读取 U2 密钥时，不应该展示 U2F 失败的错误信息

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.6
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.6
 - @alicloud/console-fetcher@2.0.0-alpha.6

zhenlan-zhao committed:
FEAT: 增加埋点 & 重新请求被风控接口带上 verifyType & 绑定 MFA 时增加跳过选项

zhenlan-zhao committed:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.41-alpha.3
 - @alicloud/console-base-global@1.1.97-alpha.3
 - @alicloud/console-base-log-sls@1.2.30-alpha.3
 - @alicloud/console-base-messenger@1.3.88-alpha.3
 - @alicloud/console-fetcher-basic@1.5.61-alpha.3
 - @alicloud/console-fetcher-interceptor-arms@1.1.82-alpha.3
 - @alicloud/console-fetcher-interceptor-fecs@1.1.92-alpha.3
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.88-alpha.3
 - @alicloud/console-fetcher-interceptor-req-security@1.1.82-alpha.3
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.82-alpha.3
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.85-alpha.3
 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.7
 - @alicloud/console-fetcher-interceptor-sls@1.1.91-alpha.3
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.7
 - @alicloud/console-fetcher@2.0.0-alpha.7
 - @alicloud/console-logger-sls@1.2.30-alpha.3
 - @alicloud/fetcher-demo-helpers@1.1.79-alpha.3
 - @alicloud/fetcher-fetch@1.2.79-alpha.3
 - @alicloud/fetcher-interceptor-cache-local@1.1.56-alpha.3
 - @alicloud/fetcher-interceptor-merger@1.0.72-alpha.3
 - @alicloud/fetcher-jsonp@1.1.80-alpha.3
 - @alicloud/fetcher@1.1.82-alpha.3
 - @alicloud/logger-sls@1.5.30-alpha.3

zhenlan-zhao committed:
FEAT: 处理新版主账号风控监听 iframe 消息时，JSON 解析报错的问题 & 增加对旧版主账号风控的埋点

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.8
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.8
 - @alicloud/console-fetcher@2.0.0-alpha.8

zhenlan-zhao committed:
FEAT: 增加对接口 getMfaAuthInfo 接口的埋点

zhenlan-zhao committed:
FEAT: 增加对接口 getMfaAuthInfo 接口的埋点

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.9
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.9
 - @alicloud/console-fetcher@2.0.0-alpha.9

zhenlan-zhao committed:
FEAT: 将 withCredential & riskVersion 参数集成进 console-fetcher & 细化 getMfaInfoToBind / getMfaInfoToAuth 接口的埋点

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.10
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.10
 - @alicloud/console-fetcher@2.0.0-alpha.10

zhenlan-zhao committed:
FEAT: console-fetcher 默认走老版本风控

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-proxy@2.0.0-alpha.11
 - @alicloud/console-fetcher@2.0.0-alpha.11

zhenlan-zhao committed:
FEAT：验证虚拟 MFA 时，用户名过长的样式修复

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.11
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.12
 - @alicloud/console-fetcher@2.0.0-alpha.12

zhenlan-zhao committed:
FEAT：U2F 升级到 WebAuthn

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.12
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.13
 - @alicloud/console-fetcher@2.0.0-alpha.13

zhenlan-zhao committed:
FEAT: WebAuthn 绑定 attachment 类型定义为 cross-plafform & u2f 重试切换为文字按钮

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.13
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.14
 - @alicloud/console-fetcher@2.0.0-alpha.14

zhenlan-zhao committed:
FEAT: webAuthn 绑定 U2F attestation 设置为 direct

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.14
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.15
 - @alicloud/console-fetcher@2.0.0-alpha.15

zhenlan-zhao committed:
FEAT: 新版风控增加轻量级虚商的验证链路

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.15
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.16
 - @alicloud/console-fetcher@2.0.0-alpha.16

zhenlan-zhao committed:
FEAT: mpk identity/send 接口 Content-Type 变更 & console-fetcher 不需要在 fetcherConfig 中加上 credential: include

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.16
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.17
 - @alicloud/console-fetcher@2.0.0-alpha.17

zhenlan-zhao committed:
FEAT: 轻量级虚商联路及其降级方案改造

zhenlan-zhao committed:
Publish

 - @alicloud/console-base-error-prompt-proxy@1.4.41-alpha.4
 - @alicloud/console-base-global@1.1.97-alpha.4
 - @alicloud/console-base-log-sls@1.2.30-alpha.4
 - @alicloud/console-base-messenger@1.3.88-alpha.4
 - @alicloud/console-fetcher-basic@1.5.61-alpha.4
 - @alicloud/console-fetcher-interceptor-arms@1.1.82-alpha.4
 - @alicloud/console-fetcher-interceptor-fecs@1.1.92-alpha.4
 - @alicloud/console-fetcher-interceptor-req-mock@1.1.88-alpha.4
 - @alicloud/console-fetcher-interceptor-req-security@1.1.82-alpha.4
 - @alicloud/console-fetcher-interceptor-res-biz@1.1.82-alpha.4
 - @alicloud/console-fetcher-interceptor-res-error-message@1.1.85-alpha.4
 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.17
 - @alicloud/console-fetcher-interceptor-sls@1.1.91-alpha.4
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.18
 - @alicloud/console-fetcher@2.0.0-alpha.18
 - @alicloud/console-logger-sls@1.2.30-alpha.4
 - @alicloud/fetcher-demo-helpers@1.1.79-alpha.4
 - @alicloud/fetcher-fetch@1.2.79-alpha.4
 - @alicloud/fetcher-interceptor-cache-local@1.1.56-alpha.4
 - @alicloud/fetcher-interceptor-merger@1.0.72-alpha.4
 - @alicloud/fetcher-jsonp@1.1.80-alpha.4
 - @alicloud/fetcher@1.1.82-alpha.4
 - @alicloud/logger-sls@1.5.30-alpha.4

zhenlan-zhao committed:
FEAT: 轻量级虚商的降级联路，请求参数里面不能有 riskVersion: 2.0

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.18
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.19
 - @alicloud/console-fetcher@2.0.0-alpha.19

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/aliyun/alibabacloud-console-base?pullRequest=16) <br/>All committers have signed the CLA.

zhenlan-zhao committed:
FEAT: 优化获取 umid 与 collina 的逻辑 & 风控服务域名在预发环境的调整 & 优化新版主账号风控 iframe 的高度

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-basic@1.6.0-alpha.0
 - @alicloud/console-fetcher-interceptor-req-security@1.2.0-alpha.0
 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.19
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.20
 - @alicloud/console-fetcher@2.0.0-alpha.20

zhenlan-zhao committed:
FEAT: 新版主账号弹窗高度固定为 400px

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-interceptor-res-risk@2.0.0-alpha.20
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.21
 - @alicloud/console-fetcher@2.0.0-alpha.21

zhenlan-zhao committed:
FEAT: 修复 security 请求拦截器取 umid 报错的问题

zhenlan-zhao committed:
Publish

 - @alicloud/console-fetcher-basic@1.6.0-alpha.1
 - @alicloud/console-fetcher-interceptor-req-security@1.2.0-alpha.1
 - @alicloud/console-fetcher-proxy@2.0.0-alpha.22
 - @alicloud/console-fetcher@2.0.0-alpha.22
